ECDSA-adaptor.md

```diff
@@ -0,0 +1,209 @@
+# ECDSA adaptor signatures
+
+DLCs work by conditioning a transaction on the revelation of an oracle's signature .
+Concretely the approach is to encrypt a transaction signature such that the oracle's released signature is the decryption key.
+Once the oracle has revealed a signature, either party may use it to decrypt the corresponding transaction signature and broadcast the transaction.
+By this method an oracle's view of a real world event enforces a particular distribution of funds between two parties according to their private bet.
+
+## Adaptor Signatures
+
+*Adaptor signatures* are an efficient form of verifiable signature encryption that can achieve our required structure.
+Given a anticipated signature `Y` we may encrypt a signature (for the purposes of this document, an ECDSA signature) under `Y` such that once a party learns the discrete logarithm of `Y` they can decrypt it and get a valid ECDSA signature.
+
+An interesting aspect of adaptor signatures is that they leak the decryption key (the discrete logarithm of `Y`) when they are decrypted.
```

Comment:
It might help to be a little more explicit here:
```suggestion
An interesting aspect of adaptor signatures is that the decryption key (the discrete logarithm of `Y`) can be derived if both the plaintext (the valid ECDSA signature) and the ciphertext (the adaptor signature) are known.
```